### PR TITLE
Update the introduction to the SDK

### PR DESCRIPTION
**Pull request by LiamDotPro**
Adds some extra information about where to find the SDK source files and also how to easily add it to you're own javascript based product.

### CODE REVIEW

File: docs/build/sdk/introduction.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ![sdk version](https://img.shields.io/github/v/tag/zeitgeistpm/tools?label=sdk)
 
+The Zeitgeist SDK is written in `Typescript` and can be found here [@zeitgeistpm/sdk](https://github.com/zeitgeistpm/tools/tree/main/packages/sdk) You can do the following to install it directly into your javascript based project:
+
+`npm i @zeitgeistpm/sdk` or `yarn add @zeitgeistpm/sdk`
```
Comment (maltekliemann) [inline docs/build/sdk/introduction.md:7]:
~~~suggestion
The Zeitgeist SDK is written in `Typescript` and can be found here:
[@zeitgeistpm/sdk](https://github.com/zeitgeistpm/tools/tree/main/packages/sdk).
You can do the following to install it directly into your javascript based
project:

```
npm i @zeitgeistpm/sdk` or `yarn add @zeitgeistpm/sdk
```
~~~

**Review by maltekliemann — CHANGES_REQUESTED** [review]
Thanks for contributing!

File: docs/build/sdk/introduction.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
 ![sdk version](https://img.shields.io/github/v/tag/zeitgeistpm/tools?label=sdk)
 
+The Zeitgeist SDK is written in `Typescript` and can be found here:
+[@zeitgeistpm/sdk](https://github.com/zeitgeistpm/tools/tree/main/packages/sdk).
+You can do the following to install it directly into your javascript based
+project:
+
+```
+npm i @zeitgeistpm/sdk or yarn add @zeitgeistpm/sdk
+```
```
Comment (maltekliemann) [inline docs/build/sdk/introduction.md:12]: Sorry, I'm dumb 🤦‍♂️. Didn't see the _or_ in there.

```suggestion
project: `npm i @zeitgeistpm/sdk` or `yarn add @zeitgeistpm/sdk`.

**Review by maltekliemann — CHANGES_REQUESTED** [review]

**Comment by LiamDotPro** [comment]
@maltekliemann Updated

**Review by maltekliemann — APPROVED** [review]

**Pull request merged**